### ISSUE TITLE
Accessibility: consistent icon usage

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#4387]
Per the audit in rancher/dashboard#4324, it has been identified that a pair of icons that refers to the same action but use different graphics. 

For example, on `dashboard/c/local/explorer/configmap`, the icons for *Download KubeConfig* and *Download Yaml* are markedly different; the former looking like a new document and the latter with a down arrow.

It has been suggested that we take a quick look at other icons to verify that they all comply, especially with the part of the rule that says that an icon has to always use the same keyword for its text label (`sr-only` class or `aria-label)`, for example, and icon for more options on a clusters list could use the text label ”Cluster options” on each row, but on the main layout it could say ”User options”. The keyword being the consistent use of the word ”Options” which will help the user identify that this icon is used for options. 

Solution for this issue: 

Use the same icon for the Download action and add a consistent text alternative.

**Comment by kwwii** [comment]
This effort should be included in the re-design of the icon theme for consistency. Common actions should use common icons.

**Comment by kwwii** [comment]
The download icon in the header needs to be the same as that used elsewhere

**Comment by aalves08** [comment]
Won't fix - covered by new accessibility issues
